common/tools/hextobin/package.json

```diff
@@ -15,8 +15,5 @@
   "main": "build/index.js",
   "bin": {
     "hextobin": "build/hextobin.js"
-  },
-  "devDependencies": {
-    "@types/node": "^18.7.18"
   }
```

Comment:
I wish there was a way in package.json to clearly document dependencies for a package while hoisting the version requirement to the top level. It's useful documentation to have :sob: